src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Pages/TermsPage.razor.scss

```
@@ -1,2 +1,39 @@
+@import '../../Styles/abstracts/_bit-css-variables.scss';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)